tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -486,6 +485,27 @@ def test_shape_2d(f):
     assert(f1(x1) == test_shape_1d(x1))
     assert(all(isclose(f2(x2), test_shape_2d(x2))))
 
+def test_shape_property():
+    @types('int[:]')
+    def test_shape_1d(f):
+        return f.shape[0]
+
+    @types('int[:,:]')
+    def test_shape_2d(f):
+        a = f.shape
+        return a[0], a[1]
+
+    from numpy import empty
+    f1 = epyccel(test_shape_1d)
+    f2 = epyccel(test_shape_2d)
+    n1 = randint(20)
+    n2 = randint(20)
+    n3 = randint(20)
+    x1 = empty(n1,dtype = int)
+    x2 = empty((n2,n3), dtype = int)
+    assert(f1(x1) == test_shape_1d(x1))
+    assert(all(isclose(f2(x2), test_shape_2d(x2))))
```

Comment:
Why are you using `isclose` to compare tuples of integers?

Reply:
Oops sorry, that's a mistake